fe1-web/src/features/evoting/objects/Configuration.ts

```diff
@@ -0,0 +1,96 @@
+import { PayloadAction } from '@reduxjs/toolkit';
+import { MessageRegistry } from 'core/network/jsonrpc/messages';
+import { Hash, Timestamp } from 'core/objects';
+import FeatureInterface from 'core/objects/FeatureInterface';
+import { EvotingFeature } from './Feature';
+
+export interface EvotingConfiguration {
```

Comment:
I'm wondering if this should be in the `objects` folder. Effectively, this is just a bunch of "stuff" - mostly functions - that comes from other features